Tipificar

arodymg opened the pull request.

arodymg committed:
Modifique los tipos de datos de los modelos de productos, reviews y categorias.
Tambien iba hacer un databaseinicializador pero mejor voy a meter los datos de las categorias y productos con el sql managment tool.
las imagenes les puse esos numeros nomas pa que sea mas facil

arodymg committed:
Volvi hacer cambios en los tipos de datos de los modelos ahora si bien.
Meti en el controlador de producto la variable de cantidad, y tambien lo agrege en las vistas de Create y Edit, lo hice porque en el migration no se creo esa columna en la tabla de productos.
Tambien instale Microsoft.EntityFrameworkCore.Tools para trabajar con los migration, nomas que ando aprendiendo como usarlo.

arodymg committed:
Le agrege en el Create donde se guardan las imagenes para que lo usen de referencia para lo del perfil de usuario.

mlopezrnet commented:
Falta actualizar la BD
![image](https://user-images.githubusercontent.com/72632296/98439928-8609f880-20b2-11eb-80bf-b3d8b1fe4a65.png)

arodymg committed:
Ya hice el migration, ya aprendi como hacerlos.

arodymg committed:
Hice un poco de cochinero con los migrations pero ya quedo hecho y jala las vistas.

mlopezrnet commented:
Si jala, aunque Details, Edit y Delete mandan error (supongo por que no hay productos en la BD).
![image](https://user-images.githubusercontent.com/72632296/98503154-152d2280-2211-11eb-8bf9-65626bc1a007.png)

arodymg committed:
Corregi eso que escribi mal los nombres de las llaves foraneas, y hice el migration para actualizarlo.
las ventanas abren, el create y el delete funcionan, mientras pongas una categoria que exista, me falta arreglar para que muestre la lista. El edit aun no funciona

arodymg committed:
Falta meter las imagenes del producto, falta hacer lo de la vista de las categorias, ligar las vistas

arodymg committed:
ahi ya funcionan para crear,editar productos
